src/ResXManager.View/Visuals/TranslationsView.xaml

```diff
@@ -107,14 +107,6 @@
                   <DockPanel DataContext="{Binding DataContext, RelativeSource={RelativeSource FindAncestor, AncestorType=DataGridCellsPanel}}" d:DataContext="{d:DesignInstance visuals:TranslationsViewModel}">
                     <StackPanel DockPanel.Dock="Right" VerticalAlignment="Center" Orientation="Horizontal">
                       <Decorator Width="20" />
-                      <CheckBox Content="{x:Static properties:Resources.TranslationsPrefixHeader}"

```

Comment:
I realize that there could be some opposition to removing this button. I looked into adding a button here that opens the Configuration page, just to make it easier for existing users to discover where the setting moved to. However, I wasn't able to easily figure out the best way to navigate to the page.
I justified this decision for myself by reasoning that Configuration options belong on the Configuration page (rather than mixing the configuration options across other pages). I wouldn't think that an experienced user of the application would be too lost trying to find where this option moved to.
With that being said, I am open to suggestions if removing the button completely is deemed too "breaking" of a change

Reply:
Why not just keep this, but have it also on the configuration page?